From 35c36d93f085d0ea02a2649795e3d4ca79358147 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Fri, 19 Aug 2022 09:53:06 +0300
Subject: [PATCH] [SW-86869] common: rename ppboot bringup function

This patch renames bringup ppboot function to enhance code
readibility.

Change-Id: I91ff3d6ae8057b5e05115fffc648f9dd5c60f568
---

diff --git a/app/arcboot_rom.c b/app/arcboot_rom.c
index 769f60b..daec631 100644
--- a/app/arcboot_rom.c
+++ b/app/arcboot_rom.c
@@ -367,7 +367,7 @@
 
 __weak void arcboot_rom_handle_tpm(void) {}
 
-__weak void arcboot_rom_bringup_pcie(void)
+__weak void arcboot_rom_bringup_ppboot(void)
 {
 	void (*ppboot)();
 
@@ -480,7 +480,7 @@
 	if (arcboot_rom_verify(HL_FW_BOOT_PTR_SPI))
 		execute_recovery();
 
-	arcboot_rom_bringup_pcie();
+	arcboot_rom_bringup_ppboot();
 	arcboot_rom_bringup_arcboot_ram();
 
 	return 0;
diff --git a/arch/gaudi3/arc/common.c b/arch/gaudi3/arc/common.c
index 534759f..fb8e5fa 100644
--- a/arch/gaudi3/arc/common.c
+++ b/arch/gaudi3/arc/common.c
@@ -796,7 +796,7 @@
 	}
 }
 
-void arcboot_rom_bringup_pcie(void)
+void arcboot_rom_bringup_ppboot(void)
 {
 	void (*ppboot)();
 	uint64_t *start_addr =
diff --git a/include/arcboot.h b/include/arcboot.h
index 1d8bb05..381a79b 100644
--- a/include/arcboot.h
+++ b/include/arcboot.h
@@ -159,7 +159,7 @@
 					enum arcboot_rom_bringup_proggress position);
 void arcboot_rom_status_update_location(uint8_t progress,
 					enum security_location loc);
-void arcboot_rom_bringup_pcie(void);
+void arcboot_rom_bringup_ppboot(void);
 void arcboot_rom_validation_fail(void);
 void arcboot_rom_clear_recovery_flag(void);
 void arcboot_rom_load_recovery(void);
